### ISSUE TITLE
Highlight / mark solved questions. 

### ISSUE THREAD

**Issue by shivhek25** [kumarneetesh24/OnlineJ#37]
Solved questions by an user can be highlighted or shown completed. It's difficult to keep a track of solved questions as of now.

**Comment by divyanshumehta** [comment]
@shivhek25 I guess it shows up in your profile
![screenshot from 2017-08-19 09-24-16](https://user-images.githubusercontent.com/16134827/29483266-37491d7a-84c0-11e7-9d69-13ca687f14c1.png)

**Comment by shivhek25** [comment]
@divyanshumehta Yes, it does but every time we want to check whether a problem has been solved or not, we will have to find out from another link if the submission has been made or not. Also, for those with higher number of submissions, it will only become more tedious. A simple feature where the question name or code is highlighted where it's displayed, if solved, would be facilitative.

**Comment by divyanshumehta** [comment]
Yes this can be added to display a green tick on contest page for solved question and cross for wrong submissons for a problem

Good one @shivhek25  :+1:

**Comment by shivhek25** [comment]
Thanks :)

**Comment by divyanshumehta** [comment]
@shivhek25 the issue seems good so I reopened it and would be implemented in near future i think.
